tests/integ-jobs/test_track_from_training_job.py

```diff
@@ -59,7 +60,7 @@ def wait_for_job(job, sagemaker_client):
                 sys.stdout.flush()
                 time.sleep(30)
 
-
+#@pytest.mark.skip(reason="temporary skipping for expediency")
```

Comment:
can we revert if this is no longer used?

Reply:
removed